src/audio/play.rs

```diff
@@ -0,0 +1,26 @@
+//! Provides functions used to play audio.
+
+use rodio::*;
+
+use std::io::Cursor;
+
+use super::source::Source;
+use super::output::Output;
+use super::DecoderError;
+
+/// Play a sound once.
+///
+/// This will return an Error if the loaded audio file in source could not be decoded.
+pub fn once(source: &Source, endpoint: &Output) -> Result<(), DecoderError> {
```

Comment:
I fixed how the pong example was handling this, can you take a look at it and tell me what you think?